shared/src/ledger/storage/merkle_tree.rs

```diff
@@ -376,7 +373,7 @@ impl<H: StorageHasher + Default> MerkleTree<H> {
     pub fn get_sub_tree_existence_proof(
         &self,
         keys: &[Key],
-        values: Vec<MerkleValue>,
+        values: Vec<Vec<u8>>,
```

Comment:
lgtm, just a small suggestion: maybe we could give a type alias to `Vec<u8>`? sort of like `TxBytes`, for txs